rqd/rqd/cuerqd.py

```diff
@@ -18,30 +18,30 @@
 """SYNOPSIS
```

Comment:
When using `argparse` you shouldn't have to list all options at the top of the file like this.

Pass `help=` to `add_argument` below to define the explanation for each argument.